partiql-test-data/fail/semantic/primitives/time-constructor.ion

```diff
@@ -0,0 +1,55 @@
+semantic::{
+    name: "invalid TIME second above upper bound",
+    statement: "TIME '12:59:61.0000'",
+    assert: {
+        result: SemanticError
+    },
+}
+
+semantic::{
+    name: "invalid TIME hour field above upper bound",
+    statement: "TIME '24:00:00'",
+    assert: {
+        result: SemanticError
+    },
+}
```

Comment:
The Kotlin implementation doesn't allow an hours field of 24 while PostgreSQL does allow `24:00:00`. I couldn't find anywhere in the SQL-92 spec that mentioned the limit.

Reply:
Yeah looks like it:
```
<hours value> ::= <datetime value>
...
<datetime value> ::= <unsigned integer>
```